docs/docs/flink-configuration.md

```diff
@@ -146,14 +146,16 @@ INSERT INTO tableName /*+ OPTIONS('upsert-enabled'='true') */
+| upsert-enabled                     | Table write.upsert.enabled                 | Overrides this table's write.upsert.enabled                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| overwrite-enabled                  | false                                      | Overwrite the table's data, overwrite mode shouldn't be enable when configuring to use UPSERT data stream.                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| distribution-mode                  | Table write.distribution-mode              | Overrides this table's write.distribution-mode. RANGE distribution is in experimental status.                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| range-distribution-statistics-type | Auto                                       | Range distribution data statistics collection type: Map, Sketch, Auto. Map type collects accurate sampling count for every single key. It should be used for low cardinality scenarios (like hundreds or thousands). Sketch type constructs a uniform random sampling via reservoir sampling. It fits well for high cardinality scenarios (like millions), as memory footprint is kept low. Auto starts with Maps statistics. But if cardinality is detected higher than a threshold (currently 10,000), statistics are automatically switched to Sketch. |
```

Comment:
only added these two rows line 156 and 157. Rest of are formatting change due to wider description column in these two rows.

Reply:
The description column is already very long for this row. maybe we can add more explanation in the `flink-writes` doc page.

Reply:
Maybe decrease the length a bit then.  Just list the possible values - or only with a single sentence each -, and link the appropriate paragraph on the `flink-writes` page.

Reply:
done